### ISSUE TITLE
Webpage do not render correctly. Biodosetools do not work.

### ISSUE THREAD

**Issue by Gaetan-IRSN** [biodosetools-team/biodosetools#4]
### Bug report/feedback

- Assessment: Github version appears not effective
- Module: Master (3.0.0) or Dev Branch (3.1.0)

Installing in R using: 
install.packages("devtools")
devtools::install_github("biodosimetry-uab/biodosetools")
Installation process seems to run correctly, no missing package or dependencies or errors are notified during the process
Running BiodoseTools : runApp()
Local session launch under local host (127.0.0.1), but webpage do not appear to render correctly:

1. On the left menu : all appear even if only Dicentrics is select on the upper scrolling menu:

![Capture-20200208180735-930x1025](https://user-images.githubusercontent.com/60822731/74089172-b3336700-4a9e-11ea-83f6-e2ea5dd63846.png)
 

2. Selecting the first Fitting in the left menu: 

- In Data input options frame: All the options appear together : custom Table or File input. The selection of "Load data from file" does change anything.

- In Data input frame: No table render or even appear when changing nb of dose of max nb of dicentrics, or when loading input file.. 

- Can not do anything.

![Capture-20200208181907-930x1025](https://user-images.githubusercontent.com/60822731/74089336-70728e80-4aa0-11ea-9c50-b03c93b1458c.png)

**Please attach an image if it helps to visualize the problem.**

**Comment by aldomann** [comment]
Thanks Gaëtan for reporting this. The issue comes from one of the Shiny packages, in particular by `shinyWidgets`.

To solve this you'll need to downgrade to the latest tested version:
```r
devtools::install_version("shinyWidgets", version = "0.4.9", repos = "http://cran.us.r-project.org")
```

Notice that bsShiny has also had a minor (non-tested) update, so until it's fully tested, I recommend running

```r
devtools::install_version("bs4Dash", version = "0.4.0", repos = "http://cran.us.r-project.org")
```

as well.

These bugs will be hopefully be addressed by v3.2.0 after finishing the migration of calculation functions to be bundled by the R package instead of being just a part of the Shiny App being done on the dev branch (v3.1.0).

**Comment by Gaetan-IRSN** [comment]
Thanks Alfredo. The downgrade of the 2 shiny packages solved the issue for the 3.0.0 and 3.1.0.

**Comment by aldomann** [comment]
Fixed by https://github.com/biodosetools-team/biodosetools/commit/bae6366a0ecbcf72f38c2334e65d03af75595a5b.
